lib/yang_wrappers.c

```diff
@@ -1127,3 +1133,31 @@ void yang_str2mac(const char *value, struct ethaddr *mac)
 {
     (void)prefix_str2mac(value, mac);
 }
+
+char *yang_nexthop_type2str(char *type_str, int len, uint32_t ntype)
```

Comment:
not just `char *yang_nexthop_type2str(uint32_t ntype)`? and directly return the type as a string without temporary strlcpy? or is here any reason for that?

Reply:
The objective is the caller can cache the returned string.

Reply:
@ton31337 I have removed `strlcpy`and return string literal.